astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -14,39 +14,80 @@ This page contains notes on changes in recent versions of FusionAuth that requir
 
 ## Version 1.99.9
 
-Version `1.99.9` does not *require* any theme changes. There are new fields, for example `tenant.passwordEnabled`, that can be used to hide the password field on the hosted login page when this value is set to `true`.
+Version `1.99.9` does not *require* any theme changes. User passwords are optional in version `1.99.9` at the tenant level.
+
+<Aside type="caution">
```

Comment:
Remove this Aside now that you can change the optional nature of passwords.

Reply:
OK. was going to warn them anyways (even though it's their form choice), but I don't think here is the place to do that now anyways. Removed.